templates/zerver/help/emoji-reactions.md

```diff
@@ -22,22 +25,75 @@ underneath the message.
 1. Select an emoji. Type to search, use the arrow keys, or click on an emoji
    with your mouse.
 
+!!! tip ""
+    To add multiple reactions without closing the emoji picker, hold the
+    <kbd>Shift</kbd> key while selecting emoji.
+
+{tab|mobile}
+
+1. Long press on a message.
+
+1. Select **Add a reaction** in the menu that appears.
+
+1. Select an emoji. Type to search, or tap the emoji you'd like to use.
+
 {end_tabs}
 
-If someone has already added a reaction, you can just click or tap on it to
-second the reaction.
+## Add or remove an existing reaction
+
+{start_tabs}
+
+{tab|desktop-web}
+
+1. Click on an existing reaction to add or remove your vote.
```

Comment:
Not sure if the following would be an improvement, but how about:

```suggestion
1. Click on an existing emoji response to add or remove your reaction.
```

Reply:
Mmm, I think I'd rather not introduce "response" as additional terminology, but adding "emoji" for clarity seems good.

Reply:
So:

> 1. Click on an existing emoji reaction to add or remove your reaction.

(and the parallel change below)